pyuploadcare/__init__.py

```diff
@@ -15,7 +15,7 @@
 
 from __future__ import unicode_literals
 
-__version__ = '2.1.0'
```

Comment:
this is not the only place for bumping package version.
let's do that in separate PR or during release altogether.